Document default resource quota of registered hosts (#3872)
theforeman/foreman-documentation: guides/common/modules/con_limiting-host-resources.adoc

= Limiting host resources

You can use the Resource Quota plugin to limit users' access to host resources during host provisioning.
Each host is assigned to exactly one resource quota.

> By default, {Project} assigns the `Unassigned` resource quota to hosts during the global host registration workflow.

Reviewer: ```suggestion
By default, {Project} assigns the `Unassigned` resource quota to hosts during global registration.
```
I hope this might suffice :)

> By default, {Project} assigns existing hosts and hosts during the global host registration workflow to the `Unassigned` resource quota.

Reviewer 2: Should this be:
```suggestion
By default, {Project} assigns existing hosts and hosts registered during the global host registration workflow to the `Unassigned` resource quota.
```

Reviewer 3: ```suggestion
By default, {Project} assigns hosts registered during the global host registration workflow to the `Unassigned` resource quota.
```

If you deploy a host, you can choose the quota. What you probably mean is that it automatically assigned 'Unassigned' at plugin installation?

> This means that users did not assign the host to a specific pool of resources.

Reviewer 4: ```suggestion
This means that the user did not assign the host to a specific pool of resources.
```

Author: Applied.
